### ISSUE TITLE
Adding language localisation options and custom classes

### ISSUE THREAD

**Issue by zipper** [ganlanyuan/tiny-slider#153]
Although I like your carousel a lot, there are two things I'm missing.

First, there is no option for localisation. I really like `aria-label`s on items making it usable for screen readers. But it would be even better, if there's an option for translating these labels into different language by making them slider option. I also missing numbers inside carousel pagination, which I believe might be useful (although it would be possible to use CSS to get them).

Second, I would make integration easier, if there's an option to specify custom classes on some elements, eg. for navigation buttons.
